Sources/BraveWallet/Crypto/Stores/SendTokenStore.swift

```diff
@@ -232,7 +232,7 @@ public class SendTokenStore: ObservableObject, WalletObserverStore {
       decimalPoint = Int(selectedSendToken?.decimals ?? 18)
       rounded = false
     }
-    sendAmount = ((selectedSendTokenBalance ?? 0) * amount.rawValue).decimalExpansion(precisionAfterDecimalPoint: decimalPoint, rounded: rounded)
+    sendAmount = selectedSendTokenBalance == 0 ? "0" : ((selectedSendTokenBalance ?? 0) * amount.rawValue).decimalExpansion(precisionAfterDecimalPoint: decimalPoint, rounded: rounded)
```

Comment:
What about using `trimmingTrailingZeros` extension instead?
And I think we should apply this in Swap too 🙂

Reply:
updated in [a2f6916](https://github.com/brave/brave-ios/pull/8234/commits/a2f6916b1266a15ac1fa8c8b42969de2def63dae)